### PR TITLE
Added artwork for Lightning Boat.

### PR DESCRIPTION
**Pull request by Dzierzan**
image

### CODE REVIEW

File: mods/hv/rules/ships.yaml [hunk 1]
```diff
@@ -212,29 +212,54 @@ BOAT2:
...
-		RecoilRecovery: 25
-		LocalOffset: 500,0,0
+		Weapon: VoltageArc
+		LocalOffset: 0,0,100
+	AttackTurreted:
+	WithAttackAnimation:
+		Sequence: shoot
+	WithAttackOverlay:
+		Sequence: charge
+		RequiresCondition: !disabled
```
Comment (Mailaender) [inline mods/hv/rules/ships.yaml:262]: This is not yet supported.

**Comment by OcelotWalrus** [comment]
Maybe it should be able to shot down aircraft?

File: mods/hv/languages/rules/en.ftl [hunk 2]
```diff
@@ -536,15 +536,15 @@ actor-submarine2 =
...
-   .name = Lightning Boat
+   .name = Railgun Boat
    .generic-name = Boat
-   .description = A heavy boat with a lightning gun.
+   .description = A heavy boat with a rail gun.
       Strong vs Vehicles, Pods and Buildings
 
-actor-boat4 =
-   .name = Railgun Boat
+actor-lightingboat =
```
Comment (Mailaender) [inline mods/hv/languages/rules/en.ftl:544]:
```suggestion
actor-lightningboat =
```

I think lighting would be :bulb: as in illumination while lightning is :zap:

Comment (OcelotWalrus) [reply mods/hv/languages/rules/en.ftl:544]: Lighting is a form of to light, while lightning is a spark of electricity, or just "the lightning of the apartment" for example.

Comment (Dzierzan) [reply mods/hv/languages/rules/en.ftl:544]: Yes, it's a typo, gonna fix it.

**Review by OcelotWalrus — APPROVED** [review]

**Pull request merged**